RebarDistribution.py

```diff
@@ -72,9 +72,22 @@ def accept(self):
             self.ExpandingLength,
         )
 
-    def setupUi(self):
+    def reject(self):
+        # self.CustomSpacing = ""
+        pass
+
+    def setupUi(self, CustomSpacing):
         # Connect Signals and Slots
         self.form.buttonBox.accepted.connect(self.accept)
+        self.form.buttonBox.rejected.connect(self.reject)
```

Comment:
Here you may use: self.form.buttonBox.rejected.connect(lambda: self.form.close())
So, reject() funcion can be deleted

Reply:
Done.
Thanks.